packages/slice-machine/lib/builders/SliceBuilder/SideBar/index.tsx

```diff
@@ -66,46 +116,21 @@ const SideBar: React.FunctionComponent<SideBarProps> = ({
         />
       </Card>
       {!isPreviewAvailableForFramework ? (
-        <InformationBox>
-          <Text as="p" variant="xs" sx={{ mb: 2 }}>
-            Slice preview is not available for this framework
+        <>
+          <Button variant="disabled" sx={{ width: "100%", mt: 3 }}>
+            Open Slice Preview
+          </Button>
+          <Text as="p" sx={{ textAlign: "center", mt: 3, color: "#667587" }}>
```

Comment:
we don't have style variable for this color ?

Reply:
textGray!